test/vuln-regex.js

```diff
@@ -0,0 +1,42 @@
+const regexp = require('../src/regexp.js');
+const vulnRegexDetector = require('vuln-regex-detector');
+
+const promises = [];
+function findRegexps(name, obj) {
+  if (typeof obj === 'string') {
+    promises.push(testRegexp(name, obj));
+  } if (obj instanceof RegExp || obj.exec) {
+    if (obj.source) {
+      promises.push(testRegexp(name, obj.source));
+    }
+  } else if (typeof obj === 'object') {
+    for (const prop in obj) {
+      findRegexps(name + (name ? '.' : '') + prop, obj[prop]);
+    }
+  }
+}
+
+async function testRegexp(name, source) {
+  try {
+    const result = await vulnRegexDetector.test(source);
```

Comment:
@davisjam is `vuln-regex-detector` not working or is there something I have to do differently here.

It seems to just throw an `INVALID` error no matter what I give it. see [travis log](https://travis-ci.org/markedjs/marked/jobs/608399445#L215)

Reply:
This was stalling on master for awhile now 🤦‍♂ 

https://travis-ci.org/markedjs/marked/jobs/593538388
https://travis-ci.org/markedjs/marked/jobs/593123904
https://travis-ci.org/markedjs/marked/jobs/592021831
https://travis-ci.org/markedjs/marked/jobs/590123069
https://travis-ci.org/markedjs/marked/jobs/588658590

Reply:
Ah, I believe some manipulation of lab computers over the summer may have brought it down.

I don't currently have the cycles to set it up properly (e.g. on Heroku) but may be able to persuade an undergraduate to undertake that task in the Spring.

For now, perhaps it's best to disable and/or ignore it.